src/sentiment.jl

```diff
@@ -8,7 +8,10 @@ function pad_sequences(l, maxlen=500)
             push!(res, ele)
         end
         return res
-    end
+    
+    else
+        print("Not Possible to have string length greater than maximum length")
```

Comment:
Throw error instead.

Reply:
https://docs.julialang.org/en/v1/manual/control-flow/ You can read about `error` here or from offline docs.

Reply:
I have made the changes you have mentioned and updated it.